app/views/application/documentation.html.erb

```diff
@@ -0,0 +1,128 @@
+<% content_for :title do %>
+  Documentation
+<% end %>
```

Comment:
I'd call this page 'API' - describes better what we're doing, and is a more useful signpost for devs